example_cpp_smart_card_client_app/build/Makefile

```diff
@@ -21,7 +21,7 @@ include $(COMMON_DIR_PATH)/make/js_building_common.mk
 
 include $(COMMON_DIR_PATH)/js/include.mk
 
-include $(COMMON_DIR_PATH)/make/app_building_common.mk
+include $(COMMON_DIR_PATH)/make/packaging_common.mk
```

Comment:
I think you also wanted to include the rename of that file, but as far as I can see it is not part of this PR

Reply:
You're right, this was an accidental change that should've gone into a different PR.